en/docs/developer-portal/test-api.md

```diff
@@ -0,0 +1,22 @@
+# Test API
+ 
+ Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API has is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test it's functionality by providing values for the required parameters. 
+
+This guide will take you through the steps to test an API in Choreo. 
+
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click  Developer Portal link on the Choreo console header. 
+    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-small}
+2. Click **APIs**.
+3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alterntively, you can click on the API and follow step 4. 
+4. Click **Try Out** from the left panel.
+5. Select the required environment from the **Environment** dropdown. 
```

Comment:
Shall we use "list" instead of "dropdown" here to comply with the latest practices we agreed on when updating the style guide?

Reply:
done

Reply:
This does not appear changed.